scrape/scrape_test.go

```diff
@@ -2393,6 +2397,16 @@ func TestTargetScraperScrapeOK(t *testing.T) {
 					"Expected Accept header to prefer application/vnd.google.protobuf.")
 			}
 
+			contentTypes := strings.Split(accept, ",")
+			for _, ct := range contentTypes {
+				match := qValuePattern.FindStringSubmatch(ct)
+				require.Len(t, match, 3)
+				qValue, err := strconv.ParseFloat(match[1], 64)
+				require.NoError(t, err, "Error parsing q value")
+				require.GreaterOrEqual(t, qValue, float64(0))
+				require.LessOrEqual(t, qValue, float64(1))
```

Comment:
Maybe we should have the rest of the checks `three digits after the decimal point` etc.
https://github.com/elnormous/contenttype could help (as the one used in `common` doesn't seem to be aligned with the RFC)